documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -398,7 +398,22 @@ This setting is ignored and network install is disabled if `DISABLE_HDMI=1` is s
 
 In order to detect the keyboard, network install must initialise the USB controller and enumerate devices. This increases boot time by approximately 1 second so it may be advantageous to disable network install in some embedded applications.
 
-Default: `1` on Raspberry Pi 4 and Raspberry Pi 400, and `0` on Compute Module 4.
+Default: `1` on Flagship models since 4 and Keyboard models since 400; `0` on Compute Modules since 4 (including Compute Module 4S).
```

Comment:
```suggestion
Default: `1` on Flagship models since Raspberry Pi 4 and Keyboard models since 400; `0` on Compute Modules since 4 (including Compute Module 4S).
```